### PR TITLE
Refactor : convert toast to snackbar

### LINKED ISSUE #1805: Replace Toast with Snackbar
As Snackbar is the "newer" android way to show small text infos, we should replace all Toast with Snackbar.
Searching for "Toast.makeText" we have 33 occurrences.

### PR DESCRIPTION
**Pull request by harshika-arya**
Fixes #1805 
Changes : Refactored `Toast` to `Snackbar` in the project

@AndyScherzinger @tobiasKaminsky 
Please review :-)

### CODE REVIEW

**Comment by mario** [comment]
Why do we use DisplayUtils somewhere and at other places Snackbar directly?

**Comment by AndyScherzinger** [comment]
Good point @mario 

@harshika-arya could you utilize DisplayUtils.showSnackMessage where possible and if not extend DisplayUtils to offer a new, fitting showSnackMessage method? That would keep thing nicely centralized as mentioned by Mario. :)

**Comment by AndyScherzinger** [comment]
> Why do we use DisplayUtils somewhere and at other places Snackbar directly?

@mario @tobiasKaminsky I took care of @mario's review comment, s the PR should now be ready for a second review :)

**Comment by AndyScherzinger** [comment]
:+1: good to go from my pov

File: src/main/java/com/owncloud/android/utils/DisplayUtils.java [hunk 1]
```diff
@@ -1,760 +1,760 @@
...
+public class DisplayUtils {
+    private static final String TAG = DisplayUtils.class.getSimpleName();
+
+    private static final String[] sizeSuffixes = {"B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB"};
+    private static final int[] sizeScales = {0, 0, 1, 1, 1, 2, 2, 2, 2};
+    private static final int RELATIVE_THRESHOLD_WARNING = 90;
+    private static final int RELATIVE_THRESHOLD_CRITICAL = 95;
+    private static final String MIME_TYPE_UNKNOWN = "Unknown type";
+
+    private static final String HTTP_PROTOCOLL = "http://";
```
Comment (mario) [inline src/main/java/com/owncloud/android/utils/DisplayUtils.java]: Ergh, remove the double L in protocol.

File: src/main/java/com/owncloud/android/utils/DisplayUtils.java [hunk 2]
```diff
@@ -1,760 +1,760 @@
...
+    private static final String HTTPS_PROTOCOLL = "https://";
```
Comment (mario) [inline src/main/java/com/owncloud/android/utils/DisplayUtils.java]: Ergh, remove the double L in protocol.

**Comment by mario** [comment]
@AndyScherzinger two small comments, and then consider it approved 👍

File: src/main/java/com/owncloud/android/utils/DisplayUtils.java (hunk 2 above)
Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/utils/DisplayUtils.java]: DONE https://github.com/nextcloud/android/pull/2389/commits/4e99e9626e3b21cebba0bf5dce58a622d2bf2d4e

File: src/main/java/com/owncloud/android/utils/DisplayUtils.java (hunk 1 above)
Comment (AndyScherzinger) [reply src/main/java/com/owncloud/android/utils/DisplayUtils.java]: DONE https://github.com/nextcloud/android/pull/2389/commits/4e99e9626e3b21cebba0bf5dce58a622d2bf2d4e

**Comment by mario** [comment]
👍 Merge.

**Comment by AndyScherzinger** [comment]
@mario Fixed! Thanks for the hints, fixed them :)

**Comment by AndyScherzinger** [comment]
Merging. Drone will keep failing due to issues with the translated strings which will go away as soon as the transfix bot pulls in the new translations

**Comment by AndyScherzinger** [comment]
Thanks for your PR @harshika-arya :heart: This is highly appreciated :1st_place_medal: 

This change will go into 3.3.0 :rocket:

**Pull request merged**